### PR TITLE
Fixes#1341 Overview links

### LINKED ISSUE #1341: Fix links to mobile clients
Right now we only have Android. Please fix the link to the play store on the overview page.

screenshot from 2018-06-15 19-50-01

### PR DESCRIPTION
**Pull request by fragm3**
Fixes #1341
Changes: (in Overview page)

Android app link
iOs app link
Github Fossasia link
-A new page would open for Twitter, Facebook, Line, Slack, We Chat, Viber, Gitter.

Note: By mistake I removed the commits, https://github.com/fossasia/chat.susi.ai/pull/1354

Demo Link: https://pr-1341-fossasia-susi-web-chat.surge.sh

### CODE REVIEW

File: src/components/Overview/Overview.react.js [hunk 1]
```diff
@@ -147,12 +146,12 @@ class Overview extends Component{
...
                    Download the App to have access to SUSI on the go.</p>
                    <div className="description__text store_icons">
-                     <Link className="playstore"
-                       target="_blank" to="https://play.google.com/store/apps/details?id=org.fossasia.susi.ai"> Get it on Google Play
-                     </Link>
-                     <Link className="appstore"
-                       target="_blank" to="https://github.com/fossasia/susi_iOS"> Download on the App Store
-                     </Link>
+                     <a className="playstore"
+                       target="_blank" rel="noopener noreferrer" href='https://play.google.com/store/apps/details?id=org.fossasia.susi.ai'> Get it on Google Play
```
Comment (Akshat-Jain) [inline src/components/Overview/Overview.react.js]: Please correct the link to https://play.google.com/store/apps/details?id=ai.susi.

Comment (fragm3) [reply src/components/Overview/Overview.react.js]: Done, please check

**Comment by Akshat-Jain** [comment]
@fragm3 Please squash the commits. Thanks :-)

**Comment by mariobehling** [comment]
Please resolve conflicts.

**Comment by fragm3** [comment]
Link to updated commit: silent-current.surge.sh

**Pull request closed without merging**